1차/이동건_완주하지 못한 선수.py

```diff
@@ -0,0 +1,20 @@
+# 20:12 ~ 20:40 (28분)
+
+def toCntDict(arr):
+    result = {}
+    for data in arr:
+        if data not in result:
+            result.update({data:0})
+        result[data]=result[data] + 1
```

Comment:
python 기본 라이브러리 collections의 Counter 를 써보세요
from collections import Counter
iterable한 데이터에 대해 elements를 카운터해서 dict로 반환 할수 있습니다.
Counter(iterable)
```suggestion
    from collections import Counter
    result = dict(Counter(arr))
```

Reply:
![image](https://user-images.githubusercontent.com/9214362/91973724-adadaf00-ed57-11ea-821e-7a17a62f5abb.png)

적용했더니 더 느려졌어요 ㅠ